### DISCUSSION TITLE
Moving external library *within* container

### DISCUSSION THREAD

**Discussion by rlichtenwalter** [immich-app/immich#7130] [Q&A]
Hi! Immich is amazing. Thank you one and all.

I initially created an external library structure and processed over a hundred thousand pictures and videos. That of course took a *long* time.

Now my use case has changed slightly, and I'll be supporting more people on the server. If I keep my current organizational scheme, both outside the container and, more importantly, within the container, it's going to be inconsistent and messy.

Right now, I have something like

/photos/phone -> /photos/phone
/photos/dslr -> /photos/dslr

What I want to change this to is

/photos/user1/phone:/photos/user1/phone
/photos/user1/dslr:/photos/user1/dslr
/photos/user2/old:/photos/user2/old
/photos/user2/dslr:/photos/user2/dslr
etc.,

Is there any way for me to make this change (of mounted location in the container) without completely reprocessing all the files from the existing external libraries?

**Comment by aisbergde** [comment]
Maybe it could be possible, using some database operation (updating the stored path). But I did not risk.

**Comment by rlichtenwalter** [comment]
I suspected that might be the case. Makes sense. I'll just move things
around and do the reprocessing. It's not that big of a deal.

**Reply by aisbergde** [reply]
One issue will be: you have to reassign assets to albums, and you have to mark assets as "archived" again.
